mobile/package.json

```diff
@@ -40,6 +40,7 @@
     "ngx-socket-io": "^4.0.0",
     "rxjs": "~6.5.5",
     "socket.io-client": "^4.0.1",
+    "tachyons": "^4.12.0",
```

Comment:
Since this is purely for development/css-preproccessing, this should probably be a development dependency.

```
npm uninstall tachyons
npm install --save-dev tachyons
```